### DISCUSSION TITLE
Image on docker hub crashing - how do you use it?

### DISCUSSION THREAD

**Discussion by UltraBlackLinux** [eclipse-theia/theia#13466] [Q&A] [answered]
https://hub.docker.com/r/eclipse/theia
what is this image for? It just exists after a few seconds? Why even provide an image if it exits immediately? I'd love to use that because the build crashes, unable to find a package.json in the `/home/theia` directory, but I don't know what it takes to make that image work

**Comment by UltraBlackLinux** [comment, marked as answer]
found out why the build was crashing. It was copying my root partition into the dockerfile because of `COPY . .` and because I forgot to set the working directory. Please don't do this...

**Reply by msujew** [reply]
Please note that the Theia team has stopped publishing images of Theia years ago. Which is why the last update on that Docker repo is from 6 years ago. Adopters of Theia are recommended to build their own images.

**Reply by UltraBlackLinux** [reply]
well the latest build is from 6 days ago...

**Reply by msujew** [reply]
>  well the latest build is from 6 days ago...

Are you sure about that?

![grafik](https://github.com/eclipse-theia/theia/assets/4377073/8afd5268-4848-4088-a374-2787b42af49e)

**Reply by UltraBlackLinux** [reply]
you know reading is hard.
